tests/api/test_story.py

```diff
@@ -1309,6 +1307,7 @@ def test_get_stories(client, resource, uids, expected):
                 'timeline_timestamp': '2017-08-13T15:43:51Z',
                 'triggered_by_advisory': {
                     'actual_ship_date': '2017-08-01T15:43:51Z',
+                    'advisory_name': 'RHBA-2017:2251-02',
```

Comment:
@sarah256 it looks like this got accidentally put in the "Remove disgit repos from User" commit. Could you please fix it?